src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.Designer.cs

```diff
@@ -727,7 +727,7 @@ public class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to The package is deprecated..
+        ///   Looks up a localized string similar to This package version is deprecated..
```

Comment:
Change isn't related, but the designer automatically regenerated this string which somehow was missed in a previous change to the file.